app/views/categories/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<header>
+    <!-- Menu icon (Hamburger menu) -->
+    <p class="menu-icon">&#9776;</p>
+    
+    <!-- Title for the page -->
+    <h2>CATEGORIES</h2>
+    
+    <!-- Search icon using Font Awesome -->
+    <i class="fa fa-search"></i>
+</header>
+
+<div class="category-list">
+  <!-- Loop through each category in @categories -->
+  <% @categories.each do |category| %>
+    <%= link_to user_category_expenses_path(@user, category), class: "category-link" do %>
+      <!-- Category container -->
+      <div class="category">
+        <!-- Category icon -->
+        <div class="category-icon">
+          <img src="<%= category.icon %>" alt="<%= category.name %>">
+        </div>
+        
+        <!-- Category name and creation date -->
+        <div>
+          <p class="name"><%= category.name %></p>
+          <p class="expense-date"><%= category.created_at.strftime("%m/%d/%Y") %></p>
+        </div>
+        
+        <!-- Total expense amount for the category -->
+        <div class="amount">
+          <p><%= number_to_currency(category.expenses.sum(:amount)) %></p>
```

Comment:
- [ ] Right now when I add new transactions the categories index page doesn't display the total number of transactions. It shows 0.00$ all the time. Please fix it.
![image](https://github.com/MohammadYaser/budget-app/assets/91486791/32f0b973-9694-4fa0-8b1b-d28d1999db67)